<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Anand" failures="1" tests="2" name="ErrorValidation" time="28.571" errors="0" timestamp="2024-08-11T23:14:52 IST">
  <testcase classname="rahulshettyacademy.SeleniumFrameworkDesign.Tests.ErrorValidationsTest" name="LogingInWithInvalidCred" time="0.416"/>
  <testcase classname="rahulshettyacademy.SeleniumFrameworkDesign.Tests.ErrorValidationsTest" name="ProductErrorValidations" time="22.662">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rahulshettyacademy.SeleniumFrameworkDesign.Tests.ErrorValidationsTest.ProductErrorValidations(ErrorValidationsTest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- ProductErrorValidations -->
</testsuite> <!-- ErrorValidation -->
